MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationAnimatorCoordinator.java

```diff
@@ -169,10 +169,14 @@ void feedNewAccuracyRadius(float targetAccuracyRadius, boolean noAnimation) {
     }
 
     float previousAccuracyRadius = getPreviousAccuracyRadius();
-    updateAccuracyAnimators(targetAccuracyRadius, previousAccuracyRadius);
-    playAnimators(
-      noAnimation || !accuracyAnimationEnabled ? 0 : ACCURACY_RADIUS_ANIMATION_DURATION,
-      ANIMATOR_LAYER_ACCURACY);
+
+    //Draw radius only if necessary
+    if (targetAccuracyRadius != 0.0 || targetAccuracyRadius != previousAccuracyRadius) {
+      updateAccuracyAnimators(targetAccuracyRadius, previousAccuracyRadius);
+      playAnimators(
+              noAnimation || !accuracyAnimationEnabled ? 0 : ACCURACY_RADIUS_ANIMATION_DURATION,
+              ANIMATOR_LAYER_ACCURACY);
+    }
 
     this.previousAccuracyRadius = targetAccuracyRadius;
```

Comment:
nit: let's move this inside of the if check to assign the new value only when the update actually happened.

Reply:
This still seems to be outside of the if-check, not a blocker though.